helper/testhelpers/teststorage/teststorage.go

```diff
@@ -199,6 +206,18 @@ func RaftBackendSetup(conf *vault.CoreConfig, opts *vault.TestClusterOptions) {
 	}
 }
 
+func RaftBackendWithAutopilotSetup(conf *vault.CoreConfig, opts *vault.TestClusterOptions) {
+	conf.DisablePerformanceStandby = true
```

Comment:
I think it's time to stop setting DisablePerformanceStandby=true by default.  I've removed that line in my PR and the tests are still passing on ent.

Reply:
👍 done